schema/dex/view_trades_alpha.sql

```diff
@@ -227,6 +227,56 @@ FROM (
         evt_index
      FROM zeroex_v3."Exchange_evt_Fill" fills
 
+    UNION
+
+    -- dYdX Solo Margin v2
+         SELECT
+        evt_block_time AS block_time,
+        'dYdX' AS project,
+        'Solo Margin v2' AS version,
+        "takerAccountOwner" AS trader_a,
+        "makerAccountOwner" AS trader_b,
+        abs(((("takerOutputUpdate"::json->>'deltaWei')::json)->>'value')::numeric)/2 AS token_a_amount_raw, --"takerOutputNumber"
+        abs(((("takerInputUpdate"::json->>'deltaWei')::json)->>'value')::numeric)/2 AS token_b_amount_raw, --"takerInputNumber"
```

Comment:
Can be simplified as
```
abs(("takerOutputUpdate"->'deltaWei'->'value')::numeric)/2 AS token_a_amount_raw,
abs(("takerInputUpdate"->'deltaWei'->'value')::numeric)/2 AS token_b_amount_raw, 
```

The `->` operator returns json while the `->>` operator returns a string!